cpp/doxygen/Doxyfile

```diff
@@ -1156,7 +1156,7 @@ HTML_HEADER            =
 # that doxygen normally uses.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-HTML_FOOTER            =
+HTML_FOOTER            = footer.html
 
```

Comment:
You also need to blank out the filename in this line: [HTML_EXTRA_STYLESHEET  = rapids.css](https://github.com/rapidsai/cudf/blob/9f2ad9cb4fe4b31f8b14fab04e3bd768060954d9/cpp/doxygen/Doxyfile#L1184)
Since you've delete the `rapids.css`

Reply:
Thanks for the catch, David. Deleted this.